src/main.ts

```diff
@@ -887,10 +888,22 @@ export class BackupFeature extends SqlOpsFeature<undefined> {
 			);
 		};
 
+		let createSas = (ownerUri: string, blobContainerUri: string, blobContainerKey: string, storageAccountName: string): Thenable<azdata.CreateSasResponse> => {
```

Comment:
Is this something that would be used outside of MS SQL? The backup provider should be generic across ALL database types, but this seems like it's scoped to only things that support it so I'm not sure it belongs here. 

@kburtram @alanrenmsft FYI

Reply:
And a SAS isn't really specific to backups to begin with, right? That's a much more generic thing (that backups just happens to use). So I'm not even sure it should be tied to backups to begin with and instead exposed as a more generic thing on our Azure functionality.

Reply:
Create SAS will be used only for MS SQL. I'm going to create a separate provider for this RPC.